examples/client-rendering/src/routes/__generated__/Route__Root__Todos__ByStatusDecodedExtra_route.res

```diff
@@ -38,17 +38,17 @@ module Internal = {
   @live
   let makePrepareProps = (. 
     ~environment: RescriptRelay.Environment.t,
-    ~pathParams: Js.Dict.t<string>,
+    ~pathParams: Dict.t<string>,
```

Comment:
Very small thing we can fix later on when it's safe to do so - `dict<string>`.

Reply:
Oops, a typo indeed!